core/server/master/src/main/java/alluxio/master/file/meta/EmptyInode.java

```diff
@@ -0,0 +1,196 @@
+/*
+ * The Alluxio Open Foundation licenses this work under the Apache License, version 2.0
+ * (the "License"). You may not use this work except in compliance with the License, which is
+ * available at www.apache.org/licenses/LICENSE-2.0
+ *
+ * This software is distributed on an "AS IS" basis, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied, as more fully set forth in the License.
+ *
+ * See the NOTICE file distributed with this work for information regarding copyright ownership.
+ */
+
+package alluxio.master.file.meta;
+
+import alluxio.grpc.TtlAction;
+import alluxio.proto.journal.Journal;
+import alluxio.proto.meta.InodeMeta;
+import alluxio.security.authorization.AccessControlList;
+import alluxio.security.authorization.AclAction;
+import alluxio.security.authorization.AclActions;
+import alluxio.security.authorization.DefaultAccessControlList;
+import alluxio.wire.FileInfo;
+
+import com.google.common.collect.ImmutableSet;
+
+import javax.annotation.Nullable;
+import java.util.List;
+import java.util.Map;
+import java.util.Objects;
+
+/**
+ * EmptyInode only supports the set and get of inodeName.
```

Comment:
Explain in the comment why this class is needed, like when and where we need it and how it is used. You can also use `{@link ...}` and `{@code ...}` to link to where this is used. For example you should explain why we only have a name here.